test/assets/names_100.json

```diff
@@ -98,3 +98,5 @@
 {"name":{"first":"Concetta","last":"Rotondo"},"gender":"female","birthday":"1992-09-19","contact":{"address":{"street":"9 19th Ave","zip":"65580","city":"Vichy","state":"MO"},"email":["concetta.rotondo@nosql-matters.org"],"region":"573","phone":[]},"likes":[],"memberSince":"2011-11-10"}
 {"name":{"first":"Georgetta","last":"Kolding"},"gender":"female","birthday":"1958-12-20","contact":{"address":{"street":"22 290th St","zip":"44645","city":"Marshallville","state":"OH"},"email":["georgetta.kolding@nosql-matters.org","kolding@nosql-matters.org"],"region":"330","phone":["330-6429454"]},"likes":["running","boxing","reading"],"memberSince":"2011-12-08"}
 {"name":{"first":"Rickie","last":"Bakaler"},"gender":"female","birthday":"1990-09-22","contact":{"address":{"street":"20 14th St","zip":"67761","city":"Wallace","state":"KS"},"email":["rickie.bakaler@nosql-matters.org"],"region":"785","phone":["785-8179421","785-2447694"]},"likes":["biking"],"memberSince":"2008-01-11"}
+{"name":{"first":"Melanie","last":"Bochaard"},"gender":"female","birthday":"1999-09-22","contact":{"address":{"street":"24 14th St","zip":"65761","city":"Manchester","state":"XX"},"email":["mél.bochard@nosql-matters.org"],"region":"785","phone":["785-8179423","785-2447693"]},"likes":["unicode"],"memberSince":"2023-01-25"}
+{"name":{"first":"Mélanie","last":"Bochaard"},"gender":"female","birthday":"1999-09-24","contact":{"address":{"street":"22 14th St","zip":"65761","city":"Manchester","state":"XX"},"email":["mÉl.bochard@nosql-matters.org"],"region":"785","phone":["785-8179425","785-2447692"]},"likes":["unicode"],"memberSince":"2023-01-24"}
```

Comment:
The name_100.json is expected to have only 100 names so if possible I would like it to stay that way. 

Would it be possible to just create some docs (one field is enough) to test this instead?

Reply:
+1